### ISSUE TITLE
Subfolder ts-migrate-full fails

### ISSUE THREAD

**Issue by etr2460** [airbnb/ts-migrate#7]
I'm trying to slowly migrate subfolders of https://github.com/apache/incubator-superset, but they seem to fail on the `eslint-fix` step because the folder cli argument isn't there. Logs are below:

```
(venv) erik-ritters-MacBook-Pro:superset-frontend erik_ritter$ npx ts-migrate-full src/CRUD
Welcome to TS Migrate! :D

This script will migrate a frontend folder to a compiling (or almost compiling) TS project.

It is recommended that you take the following steps before continuing...

1. Make sure you have a clean git slate.
   Run `git status` to make sure you have no local changes that may get lost.
   Check in or stash your changes, then re-run this script.

2. Check out a new branch for the migration.
   For example, `git checkout -b erik_ritter--ts-migrate` if you're migrating several folders or
   `git checkout -b erik_ritter--ts-migrate-CRUD` if you're just migrating src/CRUD.

3. Make sure you're on the latest, clean master.
   `git fetch origin master && git reset --hard origin/master`

4. Make sure you have the latest npm modules installed.
   `npm install` or `yarn install`

If you need help or have feedback, please file an issue on Github!

Continue? (y/N) y
Set a custom path for the typescript compiler. (It's an optional step. Skip if you don't need it. Default path is ./node_modules/.bin/tsc.): 
Your default tsc path is ./node_modules/.bin/tsc.

[Step 1 of 4] Initializing ts-config for the "src/CRUD"...

Config file created at /Users/erik_ritter/repos/github.com/incubator-superset/superset-frontend/src/CRUD/tsconfig.json
seed isort known_third_party.............................................Passed
isort................................................(no files to check)Skipped
mypy.................................................(no files to check)Skipped
pip-compile-multi verify.............................(no files to check)Skipped
Check for added large files..............................................Passed
Check docstring is first.............................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Debug Statements (Python)............................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
black................................................(no files to check)Skipped
[detached HEAD 83f4f6aa7] [ts-migrate][CRUD] Init tsconfig.json file
 2 files changed, 66 insertions(+)
 create mode 100644 superset-frontend/src/CRUD/.eslintrc
 create mode 100644 superset-frontend/src/CRUD/tsconfig.json
/Users/erik_ritter/repos/github.com/incubator-superset/superset-frontend

[Step 2 of 4] Renaming files from JS/JSX to TS/TSX and updating project.json\...

Renaming 3 JS/JSX files in /Users/erik_ritter/repos/github.com/incubator-superset/superset-frontend/src/CRUD...
Done.
seed isort known_third_party.............................................Passed
isort................................................(no files to check)Skipped
mypy.................................................(no files to check)Skipped
pip-compile-multi verify.............................(no files to check)Skipped
Check for added large files..............................................Passed
Check docstring is first.............................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Debug Statements (Python)............................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
black................................................(no files to check)Skipped
[detached HEAD 9bf8ba6b2] [ts-migrate][CRUD] Rename files from JS/JSX to TS/TSX
 3 files changed, 0 insertions(+), 0 deletions(-)
 rename superset-frontend/src/CRUD/{Field.jsx => Field.tsx} (100%)
 rename superset-frontend/src/CRUD/{Fieldset.jsx => Fieldset.tsx} (100%)
 rename superset-frontend/src/CRUD/{utils.js => utils.ts} (100%)
/Users/erik_ritter/repos/github.com/incubator-superset/superset-frontend

[Step 3 of 4] Fixing TypeScript errors...

forkTSServer
Logs in /var/folders/z4/_qft7mg12_980qwv4bsmblg40000gn/T/ts-migrate-log-7O4Yko
TypeScript version: 3.9.7
Initialized tsserver project in 215.457ms.
Start...
[strip-ts-ignore] Plugin 1 of 12. Start...
[strip-ts-ignore] Finished in 59.768ms.
[hoist-class-statics] Plugin 2 of 12. Start...
[hoist-class-statics] Finished in 2367.860ms.
[react-props] Plugin 3 of 12. Start...
[react-props] Finished in 35.451ms.
[react-class-state] Plugin 4 of 12. Start...
[react-class-state] Finished in 2.773ms.
[react-class-lifecycle-methods] Plugin 5 of 12. Start...
[react-class-lifecycle-methods] Finished in 43.427ms.
[react-default-props] Plugin 6 of 12. Start...
[react-default-props] Finished in 12.310ms.
[react-shape] Plugin 7 of 12. Start...
[react-shape] Finished in 0.690ms.
[declare-missing-class-properties] Plugin 8 of 12. Start...
[declare-missing-class-properties] Finished in 837.995ms.
[explicit-any] Plugin 9 of 12. Start...
[explicit-any] Finished in 348.799ms.
[eslint-fix] Plugin 10 of 12. Start...
npm run ts-migrate -- migrate <folder>

Fix TypeScript errors, using codemods

Positionals:
  folder                                                     [string] [required]

Options:
  -h, --help  Show help                                                [boolean]

Missing required argument: folder
```

**Comment by Rudeg** [comment]
The ts-migrate full pipeline doesn't work with subfolders. 

There are a few options:
  1) modify tsconfig to include files only from this subfolder and convert it
  2) init new temporary tsconfig in the subfolder and us it for conversion, clean up after

Alternatively, you can use the ts-migrate cli, instead of a full migration pipeline, but it might have the same problems.

**Comment by edsrzf** [comment]
I'm going to close this in favor of #44, which I believe has been addressed by #59. Confirmation would be appreciated!
